### ISSUE TITLE
A issue about the log of build scripts .

### ISSUE THREAD

**Issue by finscn** [pixijs/pixi.js#3521]
When I run  `dist` task , the log is 
```
> pixi.js@4.3.0 build ...
> npm run dist


> pixi.js@4.3.0 predist ...
> rimraf dist/**


> pixi.js@4.3.0 dist ...
> pixify -d dist -n PIXI -o pixi -t babelify

> Built pixi.js in 20.896 seconds
> Built pixi.js in 21.91 seconds
```

See the last  line.
I think  it should be  ` Built pixi.min.js` .

.

**Comment by themoonrat** [comment]
@bigtimebuddy I know this was assigned to you, but thought i'd take a look at pixify! PR submitted; then you'd need to still do the magic of creating a new version, and pointing pixi to it

**Comment by bigtimebuddy** [comment]
On it. Thanks @themoonrat!

**Comment by finscn** [comment]
Could you divid the `dist` step  into two steps :  build pixi.js & build pixi.min.js  in   scripts of package.json ?

I create a shell script to build PIXI , but some time I just want to do something to pixi.js only (e.g. in develop/test/debug mode) .

But now I have to wait until finish pixi.min.js building ( about 20 seconds ).

**Comment by themoonrat** [comment]
If you set the command line watch flag to true, it won't build the minified version

**Comment by bigtimebuddy** [comment]
npm run dist -- --watch

This will only watch and rebuild the unminified bundle.

**Comment by finscn** [comment]
@bigtimebuddy  I use the --watch , but the shell does exit .
```
> pixi.js@4.3.0 lint /Users/rocky/DEV/@git-finscn/pixi.js
> eslint scripts src test --max-warnings 0


> pixi.js@4.3.0 predist /Users/rocky/DEV/@git-finscn/pixi.js
> rimraf dist/**


> pixi.js@4.3.0 dist /Users/rocky/DEV/@git-finscn/pixi.js
> pixify -d dist -n PIXI -o pixi -t babelify "--watch"

> 3741625 bytes written (19.51 seconds)
```

It paused at this moment, not exits ,not continues.

**Comment by finscn** [comment]
@bigtimebuddy  @themoonrat  , my shell script is like this : 
```
#!/bin/bash
if [[ "$1" == "all" ]]; then
    # lint & build pixi.js and pixi.min.js
    npm run build
    cp ./dist/pixi.js ~/DEV/www/MyGameTemplate/client/lib/
    cp ./dist/pixi.min.js ~/DEV/www/MyGameTemplate/client/lib/
    exit 0
fi

# lint & build pixi.js only
npm run lint
npm run dist -- --watch
cp ./dist/pixi.js ~/DEV/www/MyGameTemplate/client/lib/
exit 0

```

but use --watch not useful.

**Comment by bigtimebuddy** [comment]
Correct @finscn, the shell does not exit. Whenever you save a file, it will rebuild. All the files are cached in memory, so you don't have to read all the files off the disk again. This makes rebuilding with watch very fast.

**Comment by finscn** [comment]
@bigtimebuddy , what shall I do , if I want to do something like my shell script ?

**Comment by bigtimebuddy** [comment]
You should created a symlink within your project to the built file. For instance:
```bash
cd ~/DEV/www/MyGameTemplate/client/lib/
ln -s /path/to/pixi.js/dist/pixi.js pixi.js
```

**Comment by finscn** [comment]
My file need to push to my git server .  symlink is not a good choice.
Is there a parameter of  pixify  to build pixi.js only ?

**Comment by finscn** [comment]
@bigtimebuddy  ,  I found the pixify.js has  a option `compress`, but the command line no this parameter.

**Comment by bigtimebuddy** [comment]
You can created a custom build script which only builds the uncompressed output. For instance:

```js
var pixify = require('pixify');
pixify({
    output: 'pixi.js',
    name: 'PIXI',
    dest: './dist/',
    compress: false,
    transform: ['babelify']
});
```

**Comment by bigtimebuddy** [comment]
Fixed in [pixify v1.7.1](https://github.com/pixijs/pixify/releases/tag/v1.7.1)
